.github/workflows/preview.yml

```diff
@@ -1,5 +1,6 @@
 name: Build and Deploy to Netlify
 on:
+  push:
```

Comment:
Please remove or comment out push as we don't need it.

Reply:
ohh..sry about that, I was working with my phone in my bed as it's 4 a.m right now.